docs/source/basics/defining_cloud_resources.md

```diff
@@ -41,11 +56,11 @@ def inference(
 
 ## Custom Task Resource
 
-You can also arbitrarily specify task resources using `@custom_task`:
+You can also specify task resources using `@custom_task`:
```

Comment:
give min/max values here, explain different node groups and what the tresholds are, link to the API doc